app/models/user.rb

```diff
@@ -0,0 +1,16 @@
+# == Schema Information
+#
+# Table name: users
+#
+#  id         :integer          not null, primary key
+#  uid        :string           not null
```

Comment:
Should this actually be named `kerb` and not `uid` for clarity (and also for ease of not-making-typos-between-`id`-and-`uid`??